test/manual/index.html

```diff
@@ -253,6 +253,23 @@
     });
 </script>
 
+<h2>Multiple MP4 Sources</h2>
+<div id="videoPlayer"></div>
+
+<script type="text/javascript">
+   jwplayer("videoPlayer").setup({
+        width:640,
+        height:360,
+        image:"http://content.jwplatform.com/thumbs/W7bCjESz-480.jpg",
+        playlist: "https://content.jwplatform.com/feeds/DrqpQIzP.json",
+
```

Comment:
Does setup complain about the hanging comma? When using setup through the harness it throws a JSON parsing error if I do